Task5.py

```diff
@@ -0,0 +1,22 @@
+"""RegularExpressions."""
+
+
+import re
+
+text = input("Enter text: " + '\n')
+expression = re.compile(r"(\S+@\S+)")
+email = expression.findall(text)
+print(email)
+print(re.findall(r"\b\w{3,5}\b", text))
+print(re.sub(r'[^0-9 ]', '', text))
```

Comment:
I should look through it a bit more

Reply:
You should not only leave all numbers but also separate them and print. For example plan is: put whitespace as second argument (' '). Then we can replase whitespaces groups with single one. Or just leave but it wouldn't be too beautiful. 
The other solution is to use other pattern for re